src/error.rs

```diff
@@ -626,6 +683,17 @@ impl<Src, Dst: ?Sized> CastError<Src, Dst> {
     }
 }
 
+impl<Src, Dst: ?Sized + Unaligned> Into<SizeError<Src, Dst>> for CastError<Src, Dst> {
+    fn into(self) -> SizeError<Src, Dst> {
```

Comment:
Slap an `#[inline(always)]` on this — we want the effects of `unreachable_unchecked` to propagate up the call chain.

Reply:
Done